doc/Language/faq.pod6

```diff
@@ -299,6 +299,77 @@ dependencies within a single compilation unit (e.g., file) are possible
 through stubbing. Therefore another possible solution is to move
 classes into the same compilation unit.
 
+
+=head1 Common operations
+
+=head2 String: How can I parse and get a L<number|/language/numerics> from a L<string|/language/type/Str>
+
+Use the L<+ prefix|/language/operators#prefix_+>:
+
+=begin code :ok-test<dd>
+say "42.123456789123456789";  # OUTPUT: «42.123456789123456789␤»
+say +"42.4e-2";               # OUTPUT: «0.424␤»
+=end code
+
+This method can numify any string you could enter as a L<litteral number|/language/syntax#Number_literals>.
+L<val|/routine/val> routine converts it to L<allomorph|/language/glossary#Allomorph>.
+L<unival|/routine/unival> routine converts one unicode codepoint.
+
+=head2 String: How can I check if a string contains a substring and if so, how can I get indices of matches
+
+Use L<.contains|/type/Str#method_contains> or L<.indices|/type/Str#method_indices>:
+
+=begin code :ok-test<dd>
+"az and az and az again".contains("az"); # OUTPUT: «True␤»
+"az and az and az again".indices("az");  # OUTPUT: «(0 7 14)␤»
+=end code
+
+=head2 String: How can I get the hexadecimal representation of a string
+
+First convert it to a L<Blob|/type/Blob> with L<.encode|/routine/encode>.
```

Comment:
Not really needed:

    "aå«".ords.map: *.base(16) # OUTPUT: (61 E5 AB)

Reply:
I must be explicit better that the desired output is one number per byte:  `"I ❤ 🦋".ords.map:  *.base(16) # OUTPUT: (49 20 2764 20 1F98B)` like an hexadecimal decoder

Reply:
Why not one number per codepoint? If you want an hexadecimal decoder, that's what you should say in the question

Reply:
__Short:__ For each byte to take the same space on output (=> alignment)

__Long:__ To align the byte list (20 bytes per line) and get the offset of the byte so you can __reverse engineer__ a binary with the offset on the side or count by hand how mush space is occupying some string, if it is zero terminated ...

Like `hexdump` command on linux 
![hex-intro](https://user-images.githubusercontent.com/6123962/77832376-5fdfab80-7114-11ea-91e5-95b8f9a36656.png)